src/test/java/org/testeditor/fixture/core/TestDefaultTestRunReporter.java

```diff
@@ -0,0 +1,290 @@
+/*******************************************************************************
+ * Copyright (c) 2012 - 2016 Signal Iduna Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * Signal Iduna Corporation - initial API and implementation
+ * akquinet AG
+ * itemis AG
+ *******************************************************************************/
+package org.testeditor.fixture.core;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InOrder;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.testeditor.fixture.core.TestRunReporter.Position;
+import org.testeditor.fixture.core.TestRunReporter.SemanticUnit;
+
+public class TestDefaultTestRunReporter {

```

Comment:
Damn, xtend just for tests is not working see [plugin issue](https://github.com/xtext/xtext-gradle-plugin/issues/65)